code/game/objects/items/weapons/towels.dm

```diff
@@ -11,12 +11,56 @@
 	desc = "A soft cotton towel."
 	material = /decl/material/solid/organic/cloth
 
+/obj/item/towel/Initialize()
+	. = ..()
+	initialize_reagents()
+	START_PROCESSING(SSobj, src)
```

Comment:
Maybe start processing on reagent change when we actually have stuff to dry and process kill once it's all dried? Just so every towel lying around isn't taking up space in ss obj